### DISCUSSION TITLE
Table Layout: Sticky Header Bar and Column Freeze

### DISCUSSION THREAD

**Discussion by hugopvl** [directus/directus#8715] [Feature Requests]
When we have many columns and hundreds of records, we get lost very easily in the information, if it was possible for the title bar to follow the information, it would help a lot :)

![titlebar](https://user-images.githubusercontent.com/81326296/136944254-de4a1cf1-0440-460a-af9b-c03082bb3bc8.jpg)

**Comment by joselcvarela** [comment]
Maybe columns could also be fixable. Useful for horizontal scroll.

**Reply by hugopvl** [reply]
Yes, it's also a good idea :D

**Reply by benhaynes** [reply]
Yes, we should 100% have the Table Layout header become fixed/sticky. Also, I had this idea for a column right-click context menu for "freezing" columns (and other options).

![Collection-Page-Column-Context](https://user-images.githubusercontent.com/522079/136971054-d2f8ca26-7d97-40e6-8e66-d5847ba62d5a.png)

**Reply by hugopvl** [reply]
Hi Guys, 
Would this idea be too difficult to implement? I have a project where this fits perfectly :)

**Reply by rijkvanzanten** [reply]
@hugopvl It's actually not that easy! 😬 Browsers are still a little finicky when it comes to `position: sticky` in overflow scrolling elements. I'm definitely down to have this in the future, but it's a little too tricky to be able to allocate the resources right now

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
